source/documentation/reference/operational-processes.md

```diff
@@ -0,0 +1,174 @@
+# Cloud Platform Operational Processes
+
+This is a record of the operational processes that we will use to support our users and to maintain the cloud platform.
+
+## Hours of support
+
+Our hours of providing support are 10am - 5pm. During this time we will work on support requests from teams and make sure someone is available to answer questions in [`#ask-cloud-platform`](https://mojdt.slack.com/messages/C57UPMZLY/) and look at PR request notifications for the cloud-platform-environments repo on [`#cloud-platform-notify`](https://mojdt.slack.com/messages/CA5MDLM34/)
```

Comment:
Do we need to clarify that 'ask-cloud-platform' is a slack channel, and/or that users don't need to post anything in 'cloud-platform-notify' because it's an automatic notifications channel?

Reply:
fixed